File: app/src/main/java/khorunzhyicom/alex/theposter/presentation/top/TopMoviesView.java
package khorunzhyicom.alex.theposter.presentation.top;


import khorunzhyicom.alex.theposter.di.components.injector.PresentationInjector;
import khorunzhyicom.alex.theposter.presentation.common.view.DaggerView;
import khorunzhyicom.alex.theposter.presentation.common.view.PosterView;

public interface TopMoviesView extends PosterView, DaggerView<PresentationInjector> {
}

File: app/src/main/java/khorunzhyicom/alex/theposter/presentation/top/TopMoviesView.kt
package khorunzhyicom.alex.theposter.presentation.top


import khorunzhyicom.alex.theposter.presentation.activity.adapter.TabView
import khorunzhyicom.alex.theposter.presentation.common.view.ContentView
import khorunzhyicom.alex.theposter.presentation.common.view.PosterBaseView
import khorunzhyicom.alex.theposter.service.models.Movie

interface TopMoviesView : PosterBaseView, TabView, ContentView<List<Movie>>
